compiler/rustc_span/src/lib.rs

```diff
@@ -68,7 +68,7 @@ mod span_encoding;
 pub use span_encoding::{DUMMY_SP, Span};
 
 pub mod symbol;
-pub use symbol::{Symbol, sym};
+pub use symbol::{Ident, MacroRulesNormalizedIdent, Symbol, kw, sym};
```

Comment:
Sorry for the tedious review. This is the line of note. The rest is just shortening `rustc_span::symbol::` qualifiers.